rest/OrderService.js

```diff
@@ -3,7 +3,9 @@ import OrderPersistence from "../persistence/OrderPersistence.js";
 import IllegalArgumentError from "./IllegalArgumentError.js";
 
 const EMPTY_ARGUMENT_ERR = "Body contains an empty argument!";
-const INVALID_TIMESTAMP_ERR = "Timestamp create should be now!";
+const PRODUCT_NOT_ACTIVE_ERR = "Product is not active!";
+const INVALID_DELETE_AUTOMATED_ERR = "Order cannot be deleted if is automated";
+const INVALID_PRODUCT_EXIST = "Оrder cannot be created because product does not exists"
```

Comment:
We don't need such thorough messages. "Ordered product doesn't exist!" is fine. After all this will be returned by the API call and can be adapted in many ways in the client-side code.